pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/json/UserCreationPayload.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.ddp.json;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.broadinstitute.ddp.model.user.UserProfile;
+
+import java.time.LocalDate;
+
+/**
+ * Payload sent during creation of a new user
+ */
+@Data
+@AllArgsConstructor
+public class UserCreationPayload {
+    @SerializedName("studyGuid")
+    private String studyGuid;
+
+    @SerializedName("firstName")
+    private String firstName;
+
+    @SerializedName("lastName")
+    private String lastName;
+
+    @SerializedName("birthDate")
+    private LocalDate birthDate;
+
+    @SerializedName("sex")
+    private UserProfile.SexType sex;
+
+    @SerializedName("centerId")
+    private Long centerId;
```

Comment:
Why is `centerId` a `Long` here? Typically, all external identifiers (such as user guids, activity codes, instance identifiers, etc) are all opaque strings.

Reply:
Spoke with @SimoneMaiwald- this will be transitioned to an opaque string guid when Centers are implemented (like what we're doing in other objects).